src/translations/en.json

```diff
@@ -309,10 +309,6 @@
   "CHECKOUT_LASTNAME_LABEL": "Last name",
   "HERO_FIRSTNAME_PLACEHOLDER": "You",
   "PHONE_OPEN_TODAY": "Open today",
-<<<<<<< HEAD
```

Comment:
Ohh... seems like you've committed some merge conflict comments :/

Reply:
Thank you!